test/functional/tools/identifier_multiple_in_conditional.xml

```diff
@@ -0,0 +1,23 @@
+<tool id="identifier_multiple_in_conditional" name="identifier_multiple_in_conditional">
+  <command><![CDATA[
+    #for $input in $the_data.input1#
+    echo '$input.element_identifier' >> 'output1';
+    #end for#
+  ]]></command>
+  <inputs>
+    <conditional name="the_data">
```

Comment:
I guess - I think repeats might be a better second test though since now that I'm thinking about this I don't think this will work properly with repeats. I'll pull this back into WIP and re-open with a more complete fix.